File: lecture-2-Arrays/hello.c
#include <cs50.h>
// string get_string(string prompt);
#include <stdio.h>
// int printf(string format, ...);

int main(void)
{
    string name = get_string("What's your name? ");
    printf("Hello, %s\n", name);
}

// $ clang -o hello hello.c -lcs50 -> link cs50 to correctly compile and execute our code
// $ ./hello

File: lecture-6-Python/hello.py
# import cs50
from cs50 import get_string

answer = get_string("What's your name: ")
print("hello world")
print("hi, " + answer) # concatenates + answer (dynamically based on variable "answer")
print("hi, ", answer) 
print(f"hello, {answer}") # format string (f-string)

# Run Python
# $ python hello.py